components/board/numato/board_test.go

```diff
@@ -91,21 +91,25 @@ func TestNumato1(t *testing.T) {
 	ar, err := b.AnalogByName("foo")
 	test.That(t, err, test.ShouldBeNil, true)
 
-	res2, err := ar.Read(ctx, nil)
+	res2, analogRange, err := ar.Read(ctx, nil)
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, res2, test.ShouldBeLessThan, 100)
 
+	test.That(t, analogRange.Min, test.ShouldEqual, 0)
+	test.That(t, analogRange.Max, test.ShouldEqual, 5)
+	test.That(t, analogRange.StepSize, test.ShouldEqual, 4.88)
```

Comment:
I don't see where these values are coming from. How did you get 4.88?

Reply:
its a range of 5 with 10 bit resolution so `5/1024` and then `*1000` to get it in mV/bit. But I just realized that for the numato tests to actually run actual board needs to be connected and they were just being skipped when I ran them. And these values would depend on what board is plugged in so they can't really be tested here.

Reply:
Ohh, it's _millivolts_ per bit! Put in a comment explaining the units.

Reply:
(not a comment here, a comment on the other place I got confused by this, at board.go line 395.)